courses/fundamentals_of_ada/adv_240_tasking.rst

```diff
@@ -625,6 +625,61 @@ Waiting On Different Entries
        exit;
      end select;
 
+------------------------
+Protected Object Entries
+------------------------
+
+* **Special** kind of protected :ada:`procedure`
+* May use a **barrier**, that **only** allows call on a **boolean** condition
+* Barrier is **evaluated** and may be **relieved** when
+
+   - A task calls :ada:`entry`
```

Comment:
I'm not sure what "A task calls 'entry'" means.
I think you really mean is "A task calls the entry" (i.e. not the keyword, but the concept)
(I think the next line is OK - re-evaluate the guard whenever a protected object is modified)